src/requirements/production.txt

```diff
@@ -40,7 +40,7 @@ oauthlib==2.1.*
 django-jsonfallback>=2.1.2
 psycopg2-binary
 # Stripe
-stripe==2.0.*
+stripe==2.27.*
```

Comment:
Since the PR was started, Stripe release ``2.29.*``, I think we can upgrade to that right away.